recipes/qt-opcua/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set version = "6.6.2" %}
+
+package:
+  name: qt6-opcua
+  version: {{ version }}
+
+source:
+  url: https://github.com/qt/qtopcua/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 3c7210493dbdf6c7ddd3fa99fd8b53510eb903cbcd2f95571d65b20eb2bc9cb2
+  patches:
+    - fix_nonamalgamated_open62541.patch
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+  run_exports:
+    - {{ pin_subpackage('qt6-opcua', max_pin='x.x') }}

```

Comment:
is there an upstream issue / patch for reference ?
if not it may be a good idea to tell them

Reply:
I could not find any issue in the qtopcua's bugtracker and I do not have/prefer not to open a Qt Account (happy to explain this in private if you like), and from what I understand you can't submit any issue/bug without a Qt Account.